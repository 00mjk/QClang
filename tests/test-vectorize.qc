int main() {
	qubit[] a;
	qubit[] b;

	b = new qubit[](3);
	a = hadamard(b);
	return 0;
}
