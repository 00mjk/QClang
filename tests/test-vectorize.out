OPENQASM 2.0;
include "qelib1.inc";
qreg q[1];
creg c[1];
h q;
qreg constemp0_qt1[1];
qreg constemp1_qt2[1];
qreg constemp2_qt3[1];
qreg constemp3_qt4[1];
qreg constemp4_qt5[1];
qreg constemp5_qt6[1];
h constemp2_qt3;
h constemp1_qt2;
h constemp0_qt1;
